tests/ctst/world/Zenko.ts

```diff
@@ -134,47 +193,48 @@ export default class Zenko extends World {
      * @returns {undefined}
      */
     async setupEntity(entityType: string): Promise<void> {
-        const keycloakPassword = this.parameters.KeycloakPassword || '123';
-        const savedParameters = JSON.parse(JSON.stringify(this.cliOptions));
+        const defaultParameters = this.parameters;
+        const keycloakPassword = defaultParameters.KeycloakTestPassword as string || '123';
```

Comment:
We can define `KeycloakTestPassword` as string in `ZenkoWorldParameters` and then

```suggestion
        const keycloakPassword = defaultParameters.KeycloakTestPassword || '123';
```

Reply:
b7460b1c75e76aefa5df57099a8cbecb7c46e999